src/components/button/Button.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+import { IBaseActionOptions } from '../actions/Action';
+import { Tooltip } from '../tooltip/Tooltip';
+
+export class Button extends React.Component<IBaseActionOptions, any> {
+
+  static defaultProps: Partial<IBaseActionOptions> = {
+    enabled: true,
+    name: '',
+    tooltip: '',
+    primary: false,
+  };
+
+  private onClick() {
+    if (this.props.onClick) {
+      this.props.onClick();
+    }
+  }
+
+  getTemplate(buttonClass: string): JSX.Element {
+    let buttonElement: JSX.Element;
+
+    const disabled: boolean = !this.props.enabled;
+
+    if (this.props.link) {
+      const target = this.props.target ? this.props.target : '';
+      const rel = 'noopener noreferrer';
+      const buttonAttrs = { disabled, target, rel };
+      buttonElement = (
+        <a className={`${buttonClass} btn-container`}
+          href={this.props.link}
+          onClick={() => this.onClick()}
```

Comment:
You could add this to the `buttonAttrs`. `buttonAttrs` could be outside of the condition and contain only `onClick` and `disabled` at first, then you could add `target` and `rel` inside of your if.